ydb/docs/en/core/reference/ydb-cli/workload-kv.md

```diff
@@ -161,6 +163,28 @@ Parameter name | Parameter description
 ---|---
 `--len` | The size of the rows in bytes that are inserted into the table as values. Default: 8.
 
+## Upsert-seq load {#upsert-seq-kv}
+
+Upsert load variant for inserting rows by sequential keys instead of random.
+
+Sequential upserts generate a concentrated load on a table (at each point in time) instead of creating a uniform load on the entire table.
```

Comment:
```suggestion
Sequential upserts generate a concentrated load on a small number of table partitions (at each point in time) instead of creating a uniform load on the entire table.
```